table_helper.cc

```diff
@@ -129,31 +129,51 @@ future<> table_helper::setup_keyspace(cql3::query_processor& qp, service::migrat
         co_return;
     }
 
-    // FIXME: call `announce` once (collapse the calls here and in `setup_table`)
```

Comment:
> Another approach is removing table_helper completely.

+1. It was added in hope that it will be reused for many tables, but since it was not there is not point in having it.